spec/system/task_spec.rb

```diff
@@ -59,22 +60,19 @@
 
   describe 'Task編集' do
     context '正常系' do
-      xit 'Taskを編集した場合、一覧画面で編集後の内容が表示されること' do
-        # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+      it 'Taskを編集した場合、一覧画面で編集後の内容が表示されること' do
+        # FIXME: テストが失敗するので修正してください ○
         visit edit_project_task_path(project, task)
         fill_in 'Deadline', with: Time.current
         click_button 'Update Task'
         click_link 'Back'
-        expect(find('.task_list')).to have_content(Time.current.strftime('%Y-%m-%d'))
+        # expect(find('.task_list')).to have_content(Task.last.deadline.strftime('%-m/%d %-H:%M'))
+        expect(find('.task_list')).to have_content(short_time(Task.last.deadline))
```

Comment:
`last`など最後に出来たデータを前提とした記載は避けた方が良いです。
```suggestion
        expect(find('.task_list')).to have_content(short_time(Time.current))
```

Reply:
cd93611　最後にできたデータではなく、現在の時刻でtestするようにへんこうしました。